### PR TITLE
Resend implementation + analytics package

### PR DESCRIPTION
**Pull request by dchan3**

### CODE REVIEW

File: client/main.css [hunk 1]
```diff
@@ -2628,6 +2628,9 @@ margin-bottom: 0 !important;
     height: auto;
     margin: 0;
   }
+  .staff__content {
```
Comment (daveynight) [inline client/main.css]: Uhh, see line [1309](#diff-1e3eca8abd395beb426e3406e1fbfa5dR1309)

File: imports/ui/components/staff/staff.html [hunk 2]
```diff
@@ -5,13 +5,6 @@ <h2 class='general__header'>KTUH Staff</h2>
     {{ #each djs }}
       {{#if name}}
         <div class='staff__item'>
-          <a href='/profile/{{usernameOf userId}}'>
-          {{#unless photo}}
-            <img class='staff__item-photo' src='/mstile-310x310.png' />
-          {{else}}
-            <img class='staff__item-photo' src={{photo.url}} />
```
Comment (daveynight) [inline imports/ui/components/staff/staff.html:12]: Please be sure these classes are deleted from `main.css`.

File: server/send_verification_link.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      }
+      else if (moment().diff(lastSent) < 1800000) {
+        return 1;
+      }
+    }
+    else {
+      return 3;
+    }
+  }
+});
```
Comment (daveynight) [inline server/send_verification_link.js:27]: F yeah! 👍

**Review by daveyshindig — CHANGES_REQUESTED** [review]
Thank you for all your hard work, Derek! I just have a couple comments for your consideration. Check it out.

**Review by daveyshindig — APPROVED** [review]

**Pull request merged**